storm-core/test/clj/org/apache/storm/cluster_test.clj

```diff
@@ -308,11 +309,11 @@
 
 (deftest test-cluster-state-default-acls
   (testing "The default ACLs are empty."
-    (stubbing [zk/mkdirs nil
+    (stubbing [Zookeeper/mkdirs nil
```

Comment:
You have the added issue that you will probably have to use mockito to validate the calls.  If you want me to put together a pull request to your branch to fix this, because it may not be that obvious please let me know and I will.